addons/mail/models/mail_tracking_value.py

```diff
@@ -184,23 +184,21 @@ def _tracking_value_format_model(self, model):
 
         formatted = [
             {
-                'changedField': col_info['string'],
                 'id': tracking.id,
                 'fieldName': tracking.field_id.name or (tracking.field_info['name'] if tracking.field_info else 'unknown'),
-                'fieldType': col_info['type'],
-                'newValue': {
+                'fieldInfo': {
+                    'changedField': col_info['string'],
                     'currencyId': tracking.currency_id.id,
-                    'value': tracking._format_display_value(col_info['type'], new=True)[0],
-                },
-                'oldValue': {
-                    'currencyId': tracking.currency_id.id,
-                    'value': tracking._format_display_value(col_info['type'], new=False)[0],
+                    'fieldName': tracking.field_id.name or (tracking.field_info['name'] if tracking.field_info else 'unknown'),
```

Comment:
Note I don't believe this is actually used in the front-end, but we could keep it "just in case" as it could be useful for various overrides. Unsure.

Reply:
From what I read, this info is indeed not needed as technically the `fieldName` would be equivalent to `changedField` but more technical as you have the field name (not the string itself).
Unless we would want to display some more technical info somewhere, I don't see it really being used. Besides it is a simple change that could be done at any point if needed.

Reply:
Sounds good, was more thinking we could leave it because it could be useful for overrides. But you can actually just add it back fairly easily with an override here in python anyway so :shrug: 

I'll sort from the zip directly and remove this then :+1: